pages/common/dexdump.md

```diff
@@ -0,0 +1,19 @@
+# dexdump
+
+> Display information about Android DEX files.
+
+- Extract classes and methods from an APK file:
+
+`dexdump {{path/to/file.apk}}`
+
+- Display DEX files header information:
```

Comment:
Just wanted to clarify whether apk files and dex files are one and the same ? Then perhaps we should mention apk file here rather than dex files. Perhaps: "Display header information of a DEX/apk file"

Reply:
An APK file is an archive that can contain several DEX files.
I tried to clarify that.